### PR TITLE
[Serializer] Document support of PHP backed enumerations

### PR DESCRIPTION
**Pull request by 94noni**
Ref https://github.com/symfony/symfony-docs/issues/15489
PS: I do not know if we should document that this requires `PHP v8.1+` and if yes, how to document it

### CODE REVIEW

**Comment by alexislefebvre** [comment]
See this example which mention that PHP 8.0 is required:

https://github.com/symfony/symfony-docs/blob/49bbf15a61af9a4f843f316f9649a8a9377311de/messenger.rst?plain=1#L53-L56

File: components/serializer.rst [hunk 1]
```diff
@@ -894,8 +894,14 @@ The Serializer component provides several built-in normalizers:
     string (``data:...``) such that files can be embedded into serialized data.
 
 :class:`Symfony\\Component\\Serializer\\Normalizer\\DateIntervalNormalizer`
-    This normalizer converts :phpclass:`DateInterval` objects into strings.
-    By default, it uses the ``P%yY%mM%dDT%hH%iM%sS`` format.
```
Comment (alexisLefebvre) [inline components/serializer.rst]: I think this should be kept as is.

File: components/serializer.rst [hunk 2]
```diff
@@ -894,8 +894,14 @@ The Serializer component provides several built-in normalizers:
...
+    This normalizer converts a \BackedEnum objects into strings or integers.
+
+:class:`Symfony\\Component\\Serializer\\Normalizer\\BackedEnumNormalizer`
+    This normalizer converts a into strings.
```
Comment (alexisLefebvre) [inline components/serializer.rst]: Something is missing, a bad copy-paste from the line above `:class:`?

Comment (94noni) [reply components/serializer.rst]: @alexislefebvre yes perhaps a too nighty PR 😴 
will have a look at it, and also check [your comment here](https://github.com/symfony/symfony-docs/pull/17101#issuecomment-1203949449)

**Comment by 94noni** [comment]
status: needs work

**Comment by 94noni** [comment]
friendly ping @alexislefebvre as reviewer :)
status: needs review

**Review by alexislefebvre — APPROVED** [review]

**Comment by javiereguiluz** [comment]
Antoine, thanks for creating so many contributions for so many features!

**Comment by 94noni** [comment]
my pleasure :)

**Comment by alexislefebvre** [comment]
> status: needs work

I didn't knew that we were able to interact with the status through carsonbot. :flushed: I learned something new, thanks!

Should it be added to the [doc of docs](https://symfony.com/doc/current/contributing/documentation/overview.html) or kept as a secret trick? :ninja:

**Comment by 94noni** [comment]
yes it is a very intelligent bot [understanding some status request](https://github.com/symfony-tools/carsonbot/blob/9d6fd8f2f37f8f2e6e58c91d78869708eab3b47d/src/Api/Status/GitHubStatusApi.php#L11) 🤖 🧠 😄

**Comment by 94noni** [comment]
@alexislefebvre I’ve seen your edit, it is documented here https://symfony.com/doc/current/contributing/community/reviews.html

**Pull request merged**